modin/pandas/test/utils.py

```diff
@@ -1132,7 +1145,25 @@ def _csv_file_maker(
         if os.path.exists(filename) and not force:
             pass
         else:
-            df = generate_dataframe(row_size, additional_col_values)
+            dates = pandas.date_range("2000", freq="h", periods=row_size)
+            data = {
+                "col1": np.arange(row_size) * 10,
+                "col2": [str(x.date()) for x in dates],
+                "col3": np.arange(row_size) * 10,
+                "col4": [str(x.time()) for x in dates],
+                "col5": [get_random_string() for _ in range(row_size)],
+                "col6": random_state.uniform(low=0.0, high=10000.0, size=row_size),
+            }
+
+            if additional_cols_values is not None:
+                assert isinstance(additional_cols_values, dict)
+                data.update(
+                    {
+                        col_name: random_state.choice(col_values, size=row_size)
+                        for col_name, col_values in additional_cols_values.items()
+                    }
+                )
+            df = pandas.DataFrame(data)
```

Comment:
This should be placed in `generate_dataframe` func.

Reply:
Corrected.